docs/relational-databases/collations/collation-and-unicode-support.md

```diff
@@ -609,26 +609,26 @@ The following table lists the encoding storage bytes for each character range an
+-  Above the ASCII range, almost all Latin-based script, and Greek, Cyrillic, Coptic, Armenian, Hebrew, Arabic, Syriac, Tāna, and N'Ko, require 2 bytes per character in both UTF-8 and UTF-16. In these cases, there are no significant storage differences for comparable data types (for example, between using **char** or **nchar**).
 -  If it's mostly East Asian script (such as Korean, Chinese, and Japanese), each character requires 3 bytes with UTF-8 and 2 bytes with UTF-16. Using UTF-16 provides storage benefits. 
--  Characters in the range 010000–10FFFF require 4 bytes in both UTF-8 and UTF-16. In these cases, there aren't storage differences for comparable data types (for example, between using **char** or **nchar**).
+-  Characters in the range 010000–10FFFF require 4 bytes in both UTF-8 and UTF-16. In these cases, there are no storage differences for comparable data types (for example, between using **char** or **nchar**).
 
 For other considerations, see [Write International Transact-SQL Statements](../../relational-databases/collations/write-international-transact-sql-statements.md).
 
 ### <a id="converting"></a> Convert to UTF-8
-Because in [CHAR(*n*) and VARCHAR(*n*)](../../t-sql/data-types/char-and-varchar-transact-sql.md) or in [NCHAR(*n*) and NVARCHAR(*n*)](../../t-sql/data-types/nchar-and-nvarchar-transact-sql.md), the *n* defines the byte storage size, not the number of characters that can be stored, it's important to determine the data type size you must convert to, in order to avoid data truncation. 
+Because in [CHAR(*n*) and VARCHAR(*n*)](../../t-sql/data-types/char-and-varchar-transact-sql.md) or in [NCHAR(*n*) and NVARCHAR(*n*)](../../t-sql/data-types/nchar-and-nvarchar-transact-sql.md), the *n* defines the byte storage size, not the number of characters that can be stored, it is important to determine the data type size you must convert to. The characters that exceed the size are to be truncated.
 
-For example, consider a column defined as **NVARCHAR(100)** that stores 180 bytes of Japanese characters. In this example, the column data is currently encoded using UCS-2 or UTF-16, which uses 2 bytes per character. Converting the column type to **VARCHAR(200)** isn't enough to prevent data truncation, because the new data type can only store 200 bytes, but Japanese characters require 3 bytes when encoded in UTF-8. So the column must be defined as **VARCHAR(270)** to avoid data loss through data truncation.
+For example, consider a column defined as **NVARCHAR(100)** that stores 180 bytes of Japanese characters. In this example, the column data is currently encoded using UCS-2 or UTF-16, which uses 2 bytes per character. Converting the column type to **VARCHAR(200)** is not sufficient to prevent data truncation, because the new data type can only store 200 bytes, but Japanese characters require 3 bytes when encoded in UTF-8. The column must be defined as **VARCHAR(270)** to avoid data loss through data truncation.
```

Comment:
```suggestion
For example, consider a column defined as **NVARCHAR(100)** that stores 180 bytes of Japanese characters. In this example, the column data is currently encoded using UCS-2 or UTF-16, which uses 2 bytes per character. Converting the column type to **VARCHAR(200)** isn't sufficient to prevent data truncation, because the new data type can only store 200 bytes, but Japanese characters require 3 bytes when encoded in UTF-8. The column must be defined as **VARCHAR(270)** to avoid data loss through data truncation.
```